site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -297,11 +313,43 @@ const CheckListCard = ({ checklist, setChecklists, owner }) => {
         <ExportDropdown {...{ checklist }} />
       </div>
       <div className="p-4">
-        {owner && owner.first_name && owner.last_name && (
-          <span className="float-right text-gray-700">
-            by {owner.first_name} {owner.last_name}
-          </span>
-        )}
+        <div className="flex float-right text-gray-700 fit-content">
+          {[
+            owner && owner.first_name && owner.last_name && (
+              <>
+                {owner.first_name} {owner.last_name}
+              </>
+            ),
+            checklist.date_created && (
+              <>
+                {' '}
+                <Tooltip
+                  content={
+                    <>
+                      Created {dateCreatedYmdHm}
+                      <br />
+                      Updated {dateUpdatedYmdHm}
+                    </>
+                  }
+                >
+                  <time dateTime={checklist.date_created}>{dateCreatedYmd}</time>
+                </Tooltip>
+              </>
+            ),
+          ]
+            .filter((e) => e)
```

Comment:
why filter falsy values? are we expecting empty fragments at this point? it would hide any errors with the above logic, which should never render an empty div, IIUC.

Reply:
We want to display the owner's name and the creation date, joined by a bullet. But sometimes the owner hasn't entered a name in the system. I'm not sure if there will be a way for the date to be undefined in production, but it sometimes happens in development. In either case, if only one of those fields is defined, we want to display it without a bullet. The `.reduce` here is the React equivalent of `.join(' • ')`. So if we have a name and a date, we get "Alice Bobson • 2024-01-02", but if we just have a date then we get just "2024-01-02". If we didn't filter out falsies, then we would get " • 2024-01-02".